.github/workflows/dockerImage.yml

```diff
@@ -61,16 +61,17 @@ jobs:
         run: |
           echo "DOCKER_REGISTRY=$(echo "${{ matrix.registry }}/${{ github.repository }}" | awk '{print tolower($0)}')" >> $GITHUB_ENV
           echo "DOCKER_IMAGE=$(echo "${{ github.repository }}" | awk -F / '{print tolower($2)}')" >> $GITHUB_ENV
-          echo BRANCH=$(echo -n "${GITHUB_REF#refs/heads/}") >> $GITHUB_ENV
+          echo CLEAN_REF=$(echo "${GITHUB_REF_NAME#refs/heads/}") >> $GITHUB_ENV
+          echo TYPE=$(echo -n "${GITHUB_REF_TYPE}") >> $GITHUB_ENV
           echo TIME_STAMP=$(echo -n "${{ steps.time.outputs.time }}") >> $GITHUB_ENV
         shell: bash
-      - name: Set tag for develop image
-        if: endsWith(env.BRANCH, 'develop') == true
-        run: echo DOCKER_IMAGE_TAG=$(echo "develop_${{ env.TIME_STAMP }}") >> $GITHUB_ENV
+      - name: Set branch_timestamp for image from branch
+        if: ${{ env.TYPE == 'branch' }}
```

Comment:
hey @niklasb99, when I push to my feature branch, is the docker image then also build?